run/websockets/test/retry.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##
+# retry.sh
+# Provides utility function commonly needed across Cloud Build pipelines to
+# retry commands on failure.
+#
+# Usage:
+# 1. Retry single command:
+#
+# ./retry.sh "CMD"
+#
+# 2. Retry with check:
+#
+# ./retry.sh "gcloud RESOURCE EXISTS?" "gcloud ACTION"
+#
+##
+
+# Usage: try "cmd1" "cmd2"
+# If first cmd executes successfully then execute second cmd
```

Comment:
I assume there's a reason this script is duplicated(?) in this folder?

Reply:
Cloud Build can't access files outside of it's directory context.